src/seedu/addressbook/data/person/address/Address.java

```diff
@@ -0,0 +1,148 @@
+package seedu.addressbook.data.person.address;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+/**
+ * Represents a Person's address in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
+ */
+public class Address {
+
+    public static final String EXAMPLE = "123, some street, some unit, 123456";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses should be [Block], [Street], [Unit], [Postal Code]";
+    public static final String ADDRESS_VALIDATION_REGEX = ".+, .+, .+, .+";
+    public static final String ADDRESS_EXTRACT_REGEX = ", ";
+
+    private static final int ADDRESS_BLOCK_INDEX = 0;
```

Comment:
2 things here:
- Good to see you have eliminated magic numbers in the code
- However, please take a look at the coding standards and find out what is the correct convention for naming constants